packages/plugins/rudder_plugin_interface/lib/src/constants.dart

```diff
@@ -60,11 +61,17 @@ class Constants {
   static const int DEFAULT_BACK_OFF_FACTOR = 2;
   static const int DEFAULT_MAX_ATTEMPTS = 10;
   static const int DEFAULT_MAX_ITEMS = 100;
+  static const bool DEFAULT_LOCK_INTEGRATIONS_VERSION = false;
+  static const bool DEFAULT_LOCK_PLUGINS_VERSION = false;
+  static const bool DEFAULT_POLYFILL_IF_REQUIRED = true;
+  static const UaChTrackLevel DEFAULT_UACH_TRACK_LEVEL = UaChTrackLevel.none;
+  static const bool DEFAULT_USE_GLOBAL_INTEGRATIONS_CONFIG_IN_EVENTS = false;
+  static const bool DEFAULT_BUFFER_DATA_PLANE_EVENTS_UNTIL_READY = false;
+  static const int DEFAULT_DATA_PLANE_EVENTS_BUFFER_TIMEOUT =
+      10 * 1000; // 10 seconds
+  static const bool DEFAULT_USE_SERVER_SIDE_COOKIES = false;
```

Comment:
Isn't there a way to avoid redefining these defaults?
Like not send that parameter to the JS SDK bridge at all if the user does not configure it.

Reply:
Done